src/components/FileInfo/FileInfoComponent.tsx

```diff
@@ -237,6 +238,9 @@ export class FileInfoComponent extends React.Component<{
             // Here is only controls for save, no need to wait file info
             case FileInfoType.SAVE_IMAGE:
             case FileInfoType.SELECT_REGION:
+                if (!AppStore.Instance.activeFrame) {
+                    return <NonIdealState icon={"folder-open"} title={"No file loaded"} description={"Load a file using the menu"} />;
+                }
                 break;
```

Comment:
don't need `break` after return

Reply:
we still need `break` when `(!AppStore.Instance.activeFrame)` is false